pkg/capture/utils/capture_image.go

```diff
@@ -40,7 +40,8 @@ func CaptureWorkloadImage(logger *log.ZapLogger, imageVersion string, debug bool
 
 	// For testing.
 	if debug {
-		if captureWorkloadImageFromEnv := os.Getenv(captureWorkloadImageEnvKey); len(captureWorkloadImageFromEnv) != 0 {
+		captureWorkloadImageFromEnv := os.Getenv(captureWorkloadImageEnvKey)
+		if len(captureWorkloadImageFromEnv) != 0 {
```

Comment:
I realize this is pre-existing, but if you do `captureWorkloadImageFromEnv == ""`, the compiler understands what you want and will intrisify this to exactly what you have here or something better.

Reply:
Changed to use equality operator rather than checking length, thanks!